charts/dev/livebook/templates/_secrets.tpl

```diff
@@ -0,0 +1,14 @@
+{{/* Define the secrets */}}
+{{- define "livebook.secrets" -}}
+{{- $secretName := (printf "%s-secrets" (include "tc.v1.common.lib.chart.names.fullname" $)) }}
+{{- $secretKeyBase := randAlphaNum 48 | b64enc -}}
```

Comment:
```suggestion
{{/* Base64 Encryption is intended, application expects a b64 formatted value */}}
{{- $secretKeyBase := randAlphaNum 48 | b64enc -}}
```

Reply:
almost. it's "just" an encoding. ;) Will add the comment.